### PR TITLE
Make rustc dependency optional

### PR DESCRIPTION
**Pull request by messense**
This is base on #96

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -33,3 +33,4 @@ winapi = { version = "0.3", features = ["winerror"] }
 
 [features]
 tmp = ["tempdir"]
+norustc = []
```
Comment (messense) [inline Cargo.toml]: Should `norustc` on by default?

**Comment by laumann** [comment]
My preference is to have new versions out-of-the-box behave the same as previous versions if possible, so if rustc is the default now, I think it should be the default in the new version.

**Comment by laumann** [comment]
Thanks for picking this up by the way, I thought the option was a good idea but it was never moved forward.

**Comment by laumann** [comment]
Thanks!

**Pull request merged**